### PR TITLE
Update release information

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -28,24 +28,26 @@ After calling the "Open AtlasMap" and that the AtlasMap panel is opened, some co
...
 
 * Check that the version in package.json has not been published yet
-  * If already published:
-    * Upgrade the version in package.json
-    * Run 'npm install' so that the package-lock.json is updated
-    * Push changes in a PR
-    * Wait for PR to be merged
+    * If already published:
+        * Upgrade the version in package.json
+        * Upgrade the version in package-lock.json
```
Comment (lhein) [inline Contributing.md:33]: What? Why should one manually edit this automatically maintained file?

File: Contributing.md [hunk 2]
```diff
@@ -28,24 +28,26 @@ After calling the "Open AtlasMap" and that the AtlasMap panel is opened, some co
...
+        * Push changes in a PR
+        * Wait for PR to be merged
 * Check that someone listed as _submitter_ in Jenkinsfile is available
 * Create a tag
 * Push the tag to vscode-atlasmap repository, it will trigger a build after few minutes
 * Check build is working fine on [Circle CI](https://app.circleci.com/pipelines/github/jboss-fuse/vscode-atlasmap)
-* Start build on [Jenkins CI](https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/VS%20Code/job/vscode-atlasmap-release) with _publishToMarketPlace_ parameter checked
+* Start build on [Jenkins CI](https://studio-jenkins-csb-codeready.apps.ocp4.prod.psi.redhat.com/job/Fuse/job/VSCode/job/vscode-atlasmap-release/) with _publishToMarketPlace_ and _publishToOVSX_ parameter checked
 * Wait the build is waiting on step _Publish to Marketplace_
```
Comment (lhein) [inline Contributing.md:41]: Not a change from this PR but the wording is really strange and not very clear

File: Contributing.md [hunk 3]
```diff
@@ -28,24 +28,26 @@ After calling the "Open AtlasMap" and that the AtlasMap panel is opened, some co
...
+  * Go to the console log of the build and click "Proceed"
+* Wait few minutes and check that it has been published on [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=redhat.atlasmap-viewer) and [Open VSX Marketplace](https://open-vsx.org/extension/redhat/atlasmap-viewer)
 * Keep build forever for later reference and edit build information to indicate the version
 * Prepare next iteration:
-  * Upgrade the version in package.json
-  * Run 'npm install' so that the package-lock.json is updated
-  * Push changes in a PR
-  * Follow PR until it is approved/merged
+    * Upgrade the version in package.json
+    * Upgrade the version in package-lock.json
```
Comment (lhein) [inline Contributing.md:50]: same as above....the wording makes me think I have to edit that file manually which is a bad idea.

File: Contributing.md (hunk 1 above)
Comment (apupier) [reply Contributing.md:33]: it avoids blowing up the whole package-lock.json when a different npm/node version is installed

Comment (lhein) [reply Contributing.md:33]: isn't that why we switched to `npm ci` ?

Comment (lhein) [reply Contributing.md:33]: I know we had that discussion before and I am still convinced that it is fine if the package-lock changes. This is a generated file. We shouldn't have to care about it at all.

Comment (apupier) [reply Contributing.md:33]: it is a generated file used to have reproducible builds. If it is changing all the times, it is is making very hard to follow updated changes and it makes the review of PR more difficult.

`npm ci` is to have the repdocuble build. To not update the package-lock.json, just reuse what is mentioned in it (and check that it is valid with package.json)

Comment (bfitzpat) [reply Contributing.md:33]: Are there any experts on the npm/package-lock situation that we can chat with to resolve this more efficiently?

File: Contributing.md (hunk 2 above)
Comment (apupier) [reply Contributing.md:41]: I do not see how to be more precise. Can you precise which part is not clear?

File: Contributing.md (hunk 1 above)
Comment (apupier) [reply Contributing.md:33]: none that I know.

I provided an explanation of the reasons to not use `npm install` and edit the file manually.

File: Contributing.md (hunk 2 above)
Comment (bfitzpat) [reply Contributing.md:41]: I might change it to "parameters" instead of "parameter" but otherwise it reads ok to me

File: Contributing.md (hunk 1 above)
Comment (bfitzpat) [reply Contributing.md:33]: isn't package-lock automatically built? so we're not necessarily updating the version as much as providing an updated file after it gets updated by the `npm install` process?

Comment (apupier) [reply Contributing.md:33]: it is clearly not great. It remains a lot better than having thousands of lines modified to review in each PR.

I will provide the change in another PR to not block the other parts that are also important.

File: Contributing.md (hunk 2 above)
Comment (lhein) [reply Contributing.md:41]: `Wait until the build is hitting the _Publish to Marketplace_ step` would read better to me but maybe its just me. `Wait the build is waiting` sounds weird to my ears.

Comment (bfitzpat) [reply Contributing.md:41]: Sorry I honestly didn't even see that part because it wasn't part of the change.
I agree that `* Wait the build is waiting on step _Publish to Marketplace_` is a little awkward. 
I would change it slightly to `* When the build hits the _Publish to Marketplace_ step, it will wait for a response.` maybe

**Review by bfitzpat — APPROVED** [review]

**Review by lhein — APPROVED** [review]
I am still not a friend of the package-lock editing but I don't want to block you.

**Comment by apupier** [comment]
> I am still not a friend of the package-lock editing but I don't want to block you.

the package-lock has been moved out of this PR https://github.com/jboss-fuse/vscode-atlasmap/pull/618
